<launch>
  <node pkg="modulair_apps_python" name="image_browser" type="image_browser_app.py" />
</launch>